tests/e2e/logging/test_openshift-logging.py

```diff
@@ -158,6 +170,6 @@ def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentcon
         pod_obj, pvc_obj = create_pvc_and_deploymentconfig_pod
 
         # Running IO on the app_pod
-        pod_obj.run_io(storage_type='fs', size=8000)
+        pod_obj.run_io(storage_type='fs', size=6000)
```

Comment:
just curious why we decided to change the size here?

Reply:
For verification of logging, actually we dont even need to run -io, here the verification would be to just create project, inside project create pod, and verify pod-logs in EFK stack. I was unsure how will an empty PV reacts, hence wanted small IO, DOnt want to complicate much by loading the PVC.